eng/testing/performance/android_scenarios.proj

```diff
@@ -42,5 +47,11 @@
         <Command>$(Python) test.py sod --scenario-name &quot;%(Identity)&quot;</Command>
         <PostCommands>$(Python) post.py</PostCommands>
     </HelixWorkItem>
+    <HelixWorkItem Include="Startup - Android HelloWorld">
```

Comment:
You are nearly there with the proper way how to use the [XHarness SDK](https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Helix/Sdk/tools/xharness-runner/Readme.md).
You can go with what you have for now as we agreed but a gentle reminder that we will require you to switch later.

We can also do it inside of this PR if you want, it should be fairly straightforward:
1. You need to zip the `$(WorkItemDirectory)` first
2. Instead of `HelixWorkItem`, you define a `<XHarnessApkToTest Include="Startup - Android HelloWorld">`
3. Instead of `PayloadDirectory` you set `ApkPath` and point it to the zipped file from 1.
4. Instead of `PreCommands`, `Command` and `PostCommands`, you will specify just one `CustomCommands` with the commands inside

Altogether, something like this:
```xml
<XHarnessApkToTest Include="Startup - Android HelloWorld">
  <ApkPath>$(PathToTheZippedWorkItemDirectory)</ApkPath>
  <CustomCommands>
    <![CDATA[
      echo on
      set XHARNESSPATH=$(XharnessPath)            <---- this is not needed as noted above
      cd $(ScenarioDirectory)helloandroid
      copy %HELIX_CORRELATION_PAYLOAD%\HelloAndroidWithDiag.apk .
      $(Python) pre.py
      $(Python) test.py devicestartup --device-type android --package-path pub\HelloAndroidWithDiag.apk --package-name net.dot.HelloAndroid --exit-code 42 --scenario-name &quot;%(Identity)&quot;
      $(Python) post.py
    ]]>
  </CustomCommands>
</XHarnessApkToTest>
```
(you don't even have to probably use things like `&quot;` inside CDATA)